src/modules/settings/styles/_rsuite.less

```diff
@@ -1,104 +1,112 @@
-@theme: 'dark'; //set to "default" for light theme

```

Comment:
can you explain the changes in this file?

Reply:
for what ever reason `prettier --write .` updated it